lib/DnD/Draggable.js

```diff
@@ -14,8 +14,10 @@ function Draggable({ children, preview, item, onBeginDrag, onEndDrag }) {
       onBeginDrag(item, monitor);
     },
     end: (targetItem, monitor) => {
-      onEndDrag(targetItem, monitor);
-      setIsDragging(false);
+      if (!monitor.didDrop()) {
+        onEndDrag(targetItem, monitor);
+        setIsDragging(false);
+      }
```

Comment:
I've started implementing the app changes, and I've found myself inserting `setIsDragging(false)` in every `onDrop` in the app. This feels a bit smelly, maybe we could hook it into `Droppable` inside gather-ui instead?